proto/autokitteh/triggers/v1/trigger.proto

```diff
@@ -11,7 +11,7 @@ message Trigger {
   string name = 2;
   string connection_id = 3;
   string env_id = 4; // if empty, applies to all envs.
-  string event_type = 5 [(buf.validate.field).string.min_len = 1];
+  string event_type = 5;
```

Comment:
are there triggers with empty event_type?

Reply:
Yes, the convention according to Itay is that either "event_type" or "filter" is required - see the new checks in `svc.proto`.

FYI, the CLI (ak trigger create) requires "event_type" or "filter" or "data" (because the scheduler requires only "data" and automatically sets "event_type").